web/packages/teleport/src/BotInstances/Details/BotInstanceDetails.tsx

```diff
@@ -0,0 +1,146 @@
+/**
+ * Teleport
+ * Copyright (C) 2025 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import { MouseEventHandler, useCallback } from 'react';
+import { useHistory, useParams } from 'react-router';
+import styled from 'styled-components';
+
+import { Alert } from 'design/Alert/Alert';
+import Box from 'design/Box/Box';
+import { ButtonBorder } from 'design/Button/Button';
+import ButtonIcon from 'design/ButtonIcon/ButtonIcon';
+import Flex from 'design/Flex/Flex';
+import { ArrowLeft } from 'design/Icon/Icons/ArrowLeft';
+import { Indicator } from 'design/Indicator/Indicator';
+import Text from 'design/Text';
+import { HoverTooltip } from 'design/Tooltip/HoverTooltip';
+import TextEditor from 'shared/components/TextEditor/TextEditor';
+import { CopyButton } from 'shared/components/UnifiedResources/shared/CopyButton';
+
+import {
+  FeatureBox,
+  FeatureHeader,
+  FeatureHeaderTitle,
+} from 'teleport/components/Layout/Layout';
+
+import { useGetBotInstance } from '../hooks';
+
+const docsURL =
```

Comment:
```suggestion
const docsUrl =
```

I know it's confusing, but our convention is for acronyms to be all caps in Go, and first-letter caps in Typescript.

Reply:
Renamed in [b732185](https://github.com/gravitational/teleport/pull/55055/commits/b7321852790410d8f86446126354b9aed294d208).